packages/tools/package.json

```diff
@@ -26,6 +26,7 @@
     "@cardstack/plugin-utils": "0.11.9",
     "@cardstack/rendering": "0.11.9",
     "@cardstack/routing": "0.11.10",
+    "@ember/test-helpers": "^1.0.1",
```

Comment:
this should not be added as an explicit dependency

Reply:
Right, but the thing is, the latest ember-cli-qunit would've only pulled in an @ember/test-helpers version that only differs in a patch version (0.7.x).

I'll check what the deal is if I use ember-qunit directly. I'm starting to get the sense that I shouldn't ever use the `ember-cli-*` add-ons, I should just stick to the `ember-*` ones :)